ksql-serde/src/main/java/io/confluent/ksql/serde/Format.java

```diff
@@ -117,25 +106,18 @@ public void validateProperties(final Map<String, String> properties) {
    * created with that stream as its source will also have the same delimiter
    * value of {@code x}</p>
    *
+   * <p>The default implementation of this method assumes that all of the
+   * supported properties are inheritable.</p>
+   *
    * @return the set of properties that are considered "inheritable"
    */
-  public Set<String> getInheritableProperties() {
-    return inheritableProperties;
+  default Set<String> getInheritableProperties() {
```

Comment:
nit: on the javadoc

> Specifies the set of "inheritable" properties - these properties will persist across streams and tables if a sink is created from a source of the same type and does not explicitly overwrite the property.

Don't really know what 'source of the same type' means.  Do you mean the sink has the same format?

Reply:
yes - updated the javadoc to clarify this (hopefully)